### ISSUE TITLE
Issues with "Built for your org" wording

### ISSUE THREAD

**Issue by catmanjan** [OfficeDev/microsoft-teams-library-js#1797]
"org" is not a word, it should be organisation, and in Australia we don't spell organisation with a Z - can this be localized?

![7dd5acaa-41bb-40c2-bfa6-78d616648f5f](https://github.com/OfficeDev/microsoft-teams-library-js/assets/137350/ea77aa8d-d796-4328-9ca4-941ced0e07a3)

**Comment by Meghana-MSFT** [comment]
Thank you for reporting this issue, we will check this and get back to you.

**Comment by Meghana-MSFT** [comment]
@catmanjan - A case has been submitted to the localization team regarding this issue, we will keep you posted once we get an update from them.

**Comment by catmanjan** [comment]
What about “org”? Or is that unlikely to change

**Comment by Meghana-MSFT** [comment]
@catmanjan - A tickethas been submitted to get advice from the localization team regarding "Built for your org" wording and localizing to the Australian spelling "organisation" in Teams. We will keep you posted, Thank you.

**Comment by Meghana-MSFT** [comment]
@catmanjan - 

1. Switching the Teams language to English Australian, shows the "orgnaisation" spelling
![image](https://github.com/OfficeDev/microsoft-teams-library-js/assets/81138249/3fcf94dd-0048-4200-9db6-e8904a136f19)

2. Engineering team has reached out to the linguist, they recommended to use 'organisation' instead of 'org'.

**Comment by catmanjan** [comment]
@Meghana-MSFT how do I change the language in Teams?

**Comment by catmanjan** [comment]
@Meghana-MSFT shouldn't the App automatically inherit the language from the machine?

**Comment by Meghana-MSFT** [comment]
To change the language in Microsoft Teams desktop client, please click on more options(three dots) which is on the left to your profile picture at the top of the app, then select Settings -> General and go to the Language section. Choose the preferred language and restart to apply the change.

![image](https://github.com/OfficeDev/microsoft-teams-library-js/assets/81138249/77bbf8af-5c1e-45af-9ae6-01d36a4ce5d1)

**Comment by catmanjan** [comment]
@Meghana-MSFT yeah I found it, can I just say that there probably isn’t anyone on earth who has worked that out! It really needs to automatically be set to the system Timezone like every other app…

**Comment by Meghana-MSFT** [comment]
Microsoft Teams automatically detects the language of your browser or operating system where possible. If your preferred language isn't supported, Teams displays either a related language or English as the default language. 

Since this is related to Teams Product, Please check the below options
For Microsoft Teams product issues/failures your tenant Admin can reach out to [Microsoft 365 Product Support](https://docs.microsoft.com/en-us/microsoft-365/admin/contact-support-for-business-products?view=o365-worldwide&tabs=online).
For general questions about Microsoft Teams please post your question on [Microsoft Teams Community](https://answers.microsoft.com/en-us/msteams/forum).

**Comment by catmanjan** [comment]
@Meghana-MSFT then there is a bug with AU time zone because it definitely is not automatically picking it up for anyone

**Comment by catmanjan** [comment]
@Meghana-MSFT any advice?

**Comment by Meghana-MSFT** [comment]
@catmanjan - Since time zone is related to Teams Product, Please check the below options
For Microsoft Teams product issues/failures your tenant Admin can reach out to [Microsoft 365 Product Support](https://docs.microsoft.com/en-us/microsoft-365/admin/contact-support-for-business-products?view=o365-worldwide&tabs=online).
For general questions about Microsoft Teams please post your question on [Microsoft Teams Community](https://answers.microsoft.com/en-us/msteams/forum).

**Comment by catmanjan** [comment]
@Meghana-MSFT shouldn't the same localisation team you contacted above deal with this?

**Comment by Meghana-MSFT** [comment]
@catmanjan - The team we reached out to checks issues related to languages and terminology. So, we suggest you raise this on above mentioned forums.
